tests/capi/meson.build

```diff
@@ -35,7 +35,7 @@ if get_option('enable-ml-service')
   unittest_capi_service_agent_client = executable('unittest_capi_service_agent_client',
     'unittest_capi_service_agent_client.cc',
     link_with: nns_capi_service_lib,
-    dependencies: [unittest_common_dep, gdbus_gen_test_dep, lib_ml_agentd_test_dep],
+    dependencies: [unittest_common_dep, nns_capi_service_dep, gdbus_gen_test_dep, lib_ml_agentd_test_dep],
```

Comment:
nns_capi_service_dep includes link_with option (link_with: nns_capi_service_lib). how about removing link_with in unittest executable?

Reply:
Thanks! I removed the `link_with` line. PTAL